### ISSUE TITLE
Possibly unmatched examples in StaticSingleAssignmentFormer's documentation

### ISSUE THREAD

**Issue by CirQ** [soot-oss/SootUp#589]
In the examples of [StaticSingleAssignmentFormer](https://github.com/soot-oss/SootUp/blob/develop/docs/advanced-topics.md#staticsingleassignmentformer), the Jimple/SSA code seems not matched the corresponding block graphs.

Concretely, in left side, `label2` immediately follows `label3`; as for the graphs, `label2` follows the `goto` block, then to `if l3<100 goto`, which is `label1` at left. Please see if the examples are correct, thanks for your efforts!

**Comment by iscas-zac** [comment]
Agreed, except the 'follow' in OP's statement should be 'be followed by' in my opinion.
